src/components/menuBar/js/menuBarController.js

```diff
@@ -84,11 +84,11 @@ MenuBarController.prototype.enableOpenOnHover = function() {
   if (this.openOnHoverEnabled) return;
   this.openOnHoverEnabled = true;
 
-  var parentToolbar;
-  if (parentToolbar = this.parentToolbar) {
+  var parentToolbar = this.parentToolbar;
+  if (parentToolbar) {
     parentToolbar.dataset.mdRestoreStyle = parentToolbar.getAttribute('style');
     parentToolbar.style.position = 'relative';
-    parentToolbar.style.zIndex = 100;
+    parentToolbar.style.zIndex = 98; // puts it directly below the backdrop
```

Comment:
Since we set the `z-index` in the SCSS for our other components, let's do the same here and simply add a classname that updates the position/z-index.